packages/cms-lib/http.js

```diff
@@ -96,7 +96,8 @@ const getRequest = async (accountId, options) => {
 };
 
 const postRequest = async (accountId, options) => {
-  return requestPN.post(await withAuth(accountId, options));
+  const data = await withAuth(accountId, options);
+  return requestPN.post(data);
```

Comment:
Curious why this is broken out?

Reply:
Cause I was logging and forgot to change it back, thanks for catching it :)